third_party/meshroom/meshroom/nodes/ARTwin/HololensPointcloudComposer.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import print_function
+
+__version__ = "0.1"
+
+from meshroom.core import desc
+import shutil
+import glob
+import os
+import sys
+os.environ["KMP_DUPLICATE_LIB_OK"]="TRUE"
+
+# import mapper packages
+dir_path = __file__
+for i in range(6):
+    dir_path = os.path.dirname(dir_path)
+sys.path.append(dir_path)
+from src.holo.HoloIO import HoloIO
+
+class HololensPointcloudComposer(desc.Node):
+
+    category = 'ARTwin'
+    documentation = '''
+This node transform the depthmaps from hololens recording into one coordinate system.
+'''
+
+    inputs = [
+        desc.File(
+            name="recordingDir",
```

Comment:
It's just a stylistic note and maybe it cannot be done here but it would be better to have consistency in spaces around equal sign.